xauth/accounts/models.py

```diff
@@ -42,4 +42,4 @@ class SecurityQuestion(AbstractSecurityQuestion):
     class Security(AbstractSecurity):
         pass
 
-    __all__.append("Security")
+    __all__.append("Security")
```

Comment:
Please revert the unnecessary change. It's the likely source of failing lint checks as reported by CI!

Reply:
Working on it